src/pages/Dashboard/Modals/SupplyWithdraw/index.spec.tsx

```diff
@@ -83,6 +83,33 @@ describe('pages/Dashboard/SupplyWithdrawUi', () => {
   });
 
   describe('Supply form', () => {
+    it.each(['ust', 'luna'])('does not display supply tab when asset is %s', async tokenId => {
```

Comment:
We could import `DISABLED_TOKENS` here to make this test more resilant

Reply:
Ah yeah of course, good point